tests/integration/accounts/test_accounts.py

```diff
@@ -0,0 +1,133 @@
+# tests/e2e/test_storage.py
+
+from tests.common.request import (
+    payload,
+    post_request_localhost,
+    wait_for_transaction,
+    call_contract_method,
+)
+
+
+from tests.common.response import (
+    has_success_status,
+    has_error_status,
+)
+
+from tests.common.accounts import create_new_account
+
+
+def test_accounts_funding():
+    account = create_new_account()
+    new_account_address = account.address
+
+    # Test fund_account
+    fund_amount = 1000
+    fund_account_result = post_request_localhost(
+        payload("fund_account", new_account_address, fund_amount)
+    ).json()
+    assert has_success_status(fund_account_result)
+    assert "transaction_id" in fund_account_result["result"]["data"]
+    wait_for_transaction(fund_account_result["result"]["data"]["transaction_id"])
+
+    # Verify balance after funding
+    get_balance_after_fund_result = post_request_localhost(
+        payload("get_balance", new_account_address)
+    ).json()
+    assert has_success_status(get_balance_after_fund_result)
+    assert (
+        get_balance_after_fund_result["result"]["data"]["account_balance"]
+        == fund_amount
+    )
+
+    # Test fund_account with invalid address
+    invalid_address = "0xinvalid_address"
+    fund_invalid_account_result = post_request_localhost(
+        payload("fund_account", invalid_address, fund_amount)
+    ).json()
+    assert has_error_status(fund_invalid_account_result)
+    print("fund_invalid_account_result", fund_invalid_account_result)
+
+    assert (
+        "Incorrect address format. Please provide a valid address."
+        in fund_invalid_account_result["result"]["message"]
+    )
+
+    # Test get_balance with invalid address
+    get_balance_invalid_result = post_request_localhost(
+        payload("get_balance", invalid_address)
+    ).json()
+    assert has_error_status(get_balance_invalid_result)
+    assert (
+        "Incorrect address format. Please provide a valid address."
+        in fund_invalid_account_result["result"]["message"]
+    )
+
+
+def test_accounts_transfers():
+    # Setup test accounts
+    account_1 = create_new_account()
+    account_1_address = account_1.address
+
+    account_2 = create_new_account()
+    account_2_address = account_2.address
```

Comment:
could you also add a test where the second account doesn't exist yet? it should get created automatically

Reply:
These two accounts only exists in the client. Account 1 is created when the fund method is called and Account 2 is created on the transfer